diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2162f9a2b..074621638 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -9,6 +9,8 @@ project(
 	DESCRIPTION "Monado(XRT) by Collabora et al"
 	)
 
+remove_definitions(-DVK_KHR_image_format_list)
+
 # CMake 3.11 introduced CMP0072 - Prefer GLVND
 if(POLICY CMP0072)
 	cmake_policy(SET CMP0072 NEW)
diff --git a/src/xrt/auxiliary/vk/vk_bundle_init.c b/src/xrt/auxiliary/vk/vk_bundle_init.c
index 8ec5df0d9..f13fb77f6 100644
--- a/src/xrt/auxiliary/vk/vk_bundle_init.c
+++ b/src/xrt/auxiliary/vk/vk_bundle_init.c
@@ -794,12 +794,12 @@ fill_in_has_device_extensions(struct vk_bundle *vk, struct u_string_list *ext_li
 		}
 #endif // defined(VK_KHR_global_priority)
 
-#if defined(VK_KHR_image_format_list)
-		if (strcmp(ext, VK_KHR_IMAGE_FORMAT_LIST_EXTENSION_NAME) == 0) {
-			vk->has_KHR_image_format_list = true;
-			continue;
-		}
-#endif // defined(VK_KHR_image_format_list)
+// #if defined(VK_KHR_image_format_list)
+// 		if (strcmp(ext, VK_KHR_IMAGE_FORMAT_LIST_EXTENSION_NAME) == 0) {
+// 			vk->has_KHR_image_format_list = true;
+// 			continue;
+// 		}
+// #endif // defined(VK_KHR_image_format_list)
 
 #if defined(VK_KHR_maintenance1)
 		if (strcmp(ext, VK_KHR_MAINTENANCE_1_EXTENSION_NAME) == 0) {
@@ -1438,9 +1438,9 @@ vk_init_from_given(struct vk_bundle *vk,
 	}
 
 	// Vulkan does not let us read what extensions was enabled.
-	if (image_format_list_enabled) {
-		vk->has_KHR_image_format_list = image_format_list_enabled;
-	}
+	// if (image_format_list_enabled) {
+	// 	vk->has_KHR_image_format_list = image_format_list_enabled;
+	// }
 
 #ifdef VK_KHR_timeline_semaphore
 	/*
diff --git a/src/xrt/auxiliary/vk/vk_helpers.c b/src/xrt/auxiliary/vk/vk_helpers.c
index 62f505391..d10a4660a 100644
--- a/src/xrt/auxiliary/vk/vk_helpers.c
+++ b/src/xrt/auxiliary/vk/vk_helpers.c
@@ -939,23 +939,23 @@ vk_create_image_mutable_rgba(
 	VkImageCreateFlags create = VK_IMAGE_CREATE_MUTABLE_FORMAT_BIT;
 	VkBaseInStructure *next_chain = NULL;
 
-#ifdef VK_KHR_image_format_list
-	VkFormat formats[2] = {
-	    VK_FORMAT_R8G8B8A8_UNORM,
-	    VK_FORMAT_R8G8B8A8_SRGB,
-	};
-
-	VkImageFormatListCreateInfoKHR image_format_list_create_info = {
-	    .sType = VK_STRUCTURE_TYPE_IMAGE_FORMAT_LIST_CREATE_INFO_KHR,
-	    .pNext = next_chain,
-	    .viewFormatCount = ARRAY_SIZE(formats),
-	    .pViewFormats = formats,
-	};
-
-	if (vk->has_KHR_image_format_list) {
-		CHAIN(image_format_list_create_info, next_chain);
-	}
-#endif
+// #ifdef VK_KHR_image_format_list
+// 	VkFormat formats[2] = {
+// 	    VK_FORMAT_R8G8B8A8_UNORM,
+// 	    VK_FORMAT_R8G8B8A8_SRGB,
+// 	};
+
+// 	VkImageFormatListCreateInfoKHR image_format_list_create_info = {
+// 	    .sType = VK_STRUCTURE_TYPE_IMAGE_FORMAT_LIST_CREATE_INFO_KHR,
+// 	    .pNext = next_chain,
+// 	    .viewFormatCount = ARRAY_SIZE(formats),
+// 	    .pViewFormats = formats,
+// 	};
+
+// 	if (vk->has_KHR_image_format_list) {
+// 		CHAIN(image_format_list_create_info, next_chain);
+// 	}
+// #endif
 
 	return create_image_simple( //
 	    vk,                     //
diff --git a/src/xrt/compositor/client/comp_vk_glue.c b/src/xrt/compositor/client/comp_vk_glue.c
index 783e5164d..7169b5618 100644
--- a/src/xrt/compositor/client/comp_vk_glue.c
+++ b/src/xrt/compositor/client/comp_vk_glue.c
@@ -27,7 +27,7 @@ const char *xrt_gfx_vk_instance_extensions = VK_KHR_EXTERNAL_FENCE_CAPABILITIES_
 const char *xrt_gfx_vk_device_extensions = VK_KHR_DEDICATED_ALLOCATION_EXTENSION_NAME
     " " VK_KHR_EXTERNAL_FENCE_EXTENSION_NAME " " VK_KHR_EXTERNAL_MEMORY_EXTENSION_NAME
     " " VK_KHR_EXTERNAL_SEMAPHORE_EXTENSION_NAME " " VK_KHR_GET_MEMORY_REQUIREMENTS_2_EXTENSION_NAME
-    " " VK_KHR_IMAGE_FORMAT_LIST_EXTENSION_NAME
+//    " " VK_KHR_IMAGE_FORMAT_LIST_EXTENSION_NAME
 
 // Platform version of "external_memory"
 #if defined(XRT_GRAPHICS_BUFFER_HANDLE_IS_FD)
diff --git a/src/xrt/state_trackers/oxr/oxr_vulkan.c b/src/xrt/state_trackers/oxr/oxr_vulkan.c
index b919dd88e..e6da7de4a 100644
--- a/src/xrt/state_trackers/oxr/oxr_vulkan.c
+++ b/src/xrt/state_trackers/oxr/oxr_vulkan.c
@@ -166,9 +166,9 @@ static const char *optional_device_extensions[] = {
 #error "Need port!"
 #endif
 
-#ifdef VK_KHR_image_format_list
-    VK_KHR_IMAGE_FORMAT_LIST_EXTENSION_NAME,
-#endif
+// #ifdef VK_KHR_image_format_list
+//     VK_KHR_IMAGE_FORMAT_LIST_EXTENSION_NAME,
+// #endif
 #ifdef VK_KHR_timeline_semaphore
     VK_KHR_TIMELINE_SEMAPHORE_EXTENSION_NAME,
 #else
